client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-description.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+class DomainConnectAuthorizeDescription extends Component {
+	placeholder = () => {
+		return (
+			<div className="domain-connect__is-placeholder">
+				<span></span>
+				<span></span>
+				<span></span>
+			</div>
+		);
+	}
+
+	getDescription = () => {
+		const { providerId } = this.props;
+
+		switch ( providerId ) {
+			case 'g-suite':
+				return translate( 'Howdy! It looks like you want to make your domain work with the ' +
```

Comment:
Ha! Good point. Most of the strings in this this PR are actually placeholders at this point. I was going to solicit input from the super helpful Editorial Team before finalizing them.